test/ShellTests.cmake

```diff
@@ -35,24 +55,13 @@ if (UNIX)
     ##############################################################################
     #  copy test programs to test dir
     ##############################################################################
-    #shell script creates dir
-    #file (MAKE_DIRECTORY "${PROJECT_BINARY_DIR}/H5TEST/swmr_test")
     add_custom_command (
         TARGET     accum_swmr_reader
         POST_BUILD
         COMMAND    ${CMAKE_COMMAND}
         ARGS       -E copy_if_different "${HDF5_SOURCE_DIR}/bin/output_filter.sh" "${HDF5_TEST_BINARY_DIR}/H5TEST/bin/output_filter.sh"
     )
 
-    #shell script creates dir
```

Comment:
Should probably remove the commented-out bits here.

Reply:
I did. At least in my clone it is.